src/views/search/search.jsx

```diff
@@ -138,4 +141,12 @@ var Search = injectIntl(React.createClass({
     }
 }));
 
-render(<Page><Search /></Page>, document.getElementById('app'));
+var mapStateToProps = function (state) {
+    return {
+        searchTerm: state.searchTerm
```

Comment:
when I tried this locally, it turned out that calling this `searchTerm` conflicts with the `getDefaultProps` method that sets the search term initially. We should probably update the way that works, but for now, could you change this to `navigation: state.navigation`?